code/src/java/gmgen/plugin/PlayerCharacterOutput.java

```diff
@@ -106,15 +105,7 @@ public String getCR()
 		float cr = (calcCR == null) ? -1 : calcCR;
 		String retString = "";
 
-		// If the CR is a fractional CR then we convert to a 1/x format
```

Comment:
This code as shown is a bug, and we need to fix not remove.  Please see plugin.exporttokens.CRToken for how fractions are handled (negative numbers are fractions)